assets/scss/_pkg.scss

```diff
@@ -0,0 +1,7 @@
+.pkg {
```

Comment:
Where is this used?

Reply:
You can ignore that part, I accidentally pushed changes that should not be in the PR.